modules/checking-mco-node-status.adoc

```diff
@@ -0,0 +1,115 @@
+====
+
+. Check the update status of nodes in a specific machine config pool by running the following command:
++
+[source,terminal]
+----
+$ oc get machineconfignodes $(oc get machineconfignodes -o json | jq -r '.items[]|select(.spec.pool.name=="<pool_name>")|.metadata.name') <1>
+----
+<1> The name of the pool is the `MachineConfigPool` object name.
++
+.Example output
+[source,text]
+----
+NAME                          UPDATED   UPDATEPREPARED   UPDATEEXECUTED   UPDATEPOSTACTIONCOMPLETE   UPDATECOMPLETE   RESUMED
+ip-10-0-48-226.ec2.internal   True      False            False            False                      False            False
+ip-10-0-5-241.ec2.internal    True      False            False            False                      False            False
+ip-10-0-74-108.ec2.internal   True      False            False            False                      False            False
+----
+
+. Check the update status of an individual node by running the following command:
++
+[source,terminal]
+----
+$ oc describe machineconfignode/<node_name> <1>
+----
+<1> The name of the node is the `MachineConfigNode` object name.
++
+.Example output
+[source,text]
+----
+Name:         <node_name>
+Namespace:
+Labels:       <none>
+Annotations:  <none>
+API Version:  machineconfiguration.openshift.io/v1alpha1
+Kind:         MachineConfigNode
+Metadata:
+  Creation Timestamp:  2023-10-17T13:08:58Z
+  Generation:          1
+  Resource Version:    49443
+  UID:                 4bd758ab-2187-413c-ac42-882e61761b1d
+Spec:
+  Node Ref:
+    Name:         <node_name>
+  Pool:
+    Name:         master
+  ConfigVersion:
+    Desired: rendered-worker-823ff8dc2b33bf444709ed7cd2b9855b <1>
+Status:
+  Conditions:
+    Last Transition Time:  2023-10-17T13:09:02Z
+    Message:               Node has completed update to config rendered-master-cf99e619747ab19165f11e3546c71f1e
+    Reason:                NodeUpgradeComplete
+    Status:                True
+    Type:                  Updated
+    Last Transition Time:  2023-10-17T13:09:02Z
+    Message:               This node has not yet entered the UpdatePreparing phase
+    Reason:                NotYetOccured
+    Status:                False
+  Config Version:
+    Current:            rendered-worker-823ff8dc2b33bf444709ed7cd2b9855b
+    Desired:            rendered-worker-823ff8dc2b33bf444709ed7cd2b9855b <2>
+  Health:               Healthy
+  Most Recent Error:
+  Observed Generation:  3
+----
+<1> The desired configuration specified in the `spec.configversion.desired` field updates immediately when a new configuration is detected on the node.
+<2> The desired configuration specified in the `status.configversion.desired` field only updates when the new configuration is validated by the Machine Config Daemon (MCD). The MCD performs validation by checking the current phase of the update. If the update successfully passes the `UPDATEPREPARED` phase, then the status adds the new configuration.
```

Comment:
Per the ISG it's best to place "only" [directly before the word or phrase it modifies](https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only). If you mean to say "the only time `status.configversion.desired` updates is when new config is validated" I'd write it like this:
```suggestion
<2> The desired configuration specified in the `status.configversion.desired` field updates only when the new configuration is validated by the Machine Config Daemon (MCD). The MCD performs validation by checking the current phase of the update. If the update successfully passes the `UPDATEPREPARED` phase, then the status adds the new configuration.
```

But if this is meant to say "the only thing this field does is update when new config is validated", I think it's fine to keep as is!

Reply:
good catch, thanks!